src/Components/TreeComponent/package.json

```diff
@@ -4,6 +4,7 @@
     "description": "Project containing your PowerApps Component Framework (PCF) control.",
     "scripts": {
         "prepare": "cd ../../.. && husky install src/components/.husky",
+        "postinstall": "copy .\\node_modules\\antd\\dist\\antd.css .\\TreeComponent\\css",
```

Comment:
Can this be solved by importing the CSS? https://stackoverflow.com/questions/54736207/ant-design-css-not-loading-properly

Reply:
May not be as simple as that, I have tried that, and the importing was causing issues with the PCF framework which is why I had imported the CSS like this. That's not to say there isn't a better way, but I remember spending lots of time on this issue. Worth revisiting for sure, but maybe not make this part of the PR if it takes too long to fix.

Reply:
gave it a shot. looks like it works now! thanks!